TerrainMap:
    filter_radius: .5
    elevation_map_res: .1